backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -377,4 +383,24 @@ public ExternalUploadResult uploadEntities(Subject subject, Dataset dataset, Ext
 				statistic.getUnreadableDate()
 		);
 	}
+
+	public FullExecutionStatus getSingleEntityExport(Subject subject, String entity, List<Connector> sources) {
```

Comment:
Diese Anfragen sollten wahrscheinlich nicht persistiert werden oder?

Reply:
Ist es hier nicht erstmal intuitiver Konzepte entgegenzunehmen und daraus die Konnektoren zu extrahieren?

Reply:
> Diese Anfragen sollten wahrscheinlich nicht persistiert werden oder?

Guter Punkt.

> Ist es hier nicht erstmal intuitiver Konzepte entgegenzunehmen und daraus die Konnektoren zu extrahieren?

Ich denke nicht, weil es meistens wirklich um bestimmte Quellen gehen wird. Am liebsten wären mir natürlich Tables aber ich weiß nicht ob wir das mal ganz spontan mit in die Api ziehen können.